overlord/auth/auth.go

```diff
@@ -179,6 +184,7 @@ func NewUser(st *state.State, userData NewUserData) (*UserState, error) {
 		Discharges:      nil,
 		StoreMacaroon:   userData.Macaroon,
 		StoreDischarges: userData.Discharges,
+		Expiration:      userData.Expiration,
```

Comment:
it seems we need a new NewUser test to cover this bit?

Reply:
Yea very nice catch, definitely missed that one